README.md

```diff
@@ -1,2 +1,9 @@
-# JavaScript-Basic-Configuration
-Template Reppo
+# JavaScript Todo List
+
+This project counts towards Microverse graduation.
+
+## About
+
+Done for the browser, this to-do list aims a bit higher by gamifying the taks and processes, making more fun to do your daily to-dos.
+This (super) alpha release has the intent of developing the most basic mecanic of the app: The act to register tasks, complete, edit, etc.
+Despite it's still not finished, it obeys the project specifications of functionality.
```

Comment:
this README isn't complete
- add a live demo link so that incase TSE has issues testing locally, this live version might help, and also, not everyone will clone
- You might want to add screenshot also
- You might equally add howto clone the project, run it locally and author's contact
etc